From 26c2385faed0bb997368daabd4c64207b3fc01ca Mon Sep 17 00:00:00 2001
From: Fabrice Fontaine <fontaine.fabrice@gmail.com>
Date: Fri, 15 Apr 2022 18:58:49 +0200
Subject: [PATCH] gst-python: fix build on systems without C++ compiler

Fix the following build failure on systems without C++ compiler:

The following exception(s) were encountered:
Running "/home/autobuild/autobuild/instance-0/output-1/host/bin/i686-buildroot-linux-gnu-g++ --version" gave "[Errno 2] No such file or directory: '/home/autobuild/autobuild/instance-0/output-1/host/bin/i686-buildroot-linux-gnu-g++'"

Fixes:
 - http://autobuild.buildroot.org/results/eebf65036f79d21d347714d62afecd0108393308

Part-of: <https://gitlab.freedesktop.org/gstreamer/gstreamer/-/merge_requests/2200>

[Retrieved (and backported) from:
https://gitlab.freedesktop.org/gstreamer/gstreamer/-/commit/26c2385faed0bb997368daabd4c64207b3fc01ca]
Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
---
 subprojects/gst-python/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 1462604b297..c28df16192a 100644
--- a/meson.build
+++ b/meson.build
@@ -1,4 +1,4 @@
-project('gst-python', 'c', 'cpp',
+project('gst-python', 'c',
   version : '1.20.1',
   meson_version : '>= 0.60',
   default_options : [ 'warning_level=1',
-- 
GitLab

